reslib/pom.xml

```diff
@@ -47,10 +47,10 @@
 									<goal>wget</goal>
 								</goals>
 								<configuration>
-									<url>https://github.com/liferay/liferay-portal/raw/0f75db5b5c6a8a7c6b07295cc12acdf362bfb1b4/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
+									<url>https://github.com/liferay/liferay-portal/raw/6.2.x/portal-web/third-party/alloy-${alloyui.zip.version}.zip</url>
```

Comment:
@juangon, if we change the URL to use `6.2.x`, the build will break the next time they upgrade the AlloyUI zip due to the md5 not matching. It's much better to specify a commit hash (`eb2dc9930ff437e2b75477632fd06dd1b7932cf0`) so the build will always work. That's why I did [this commit](https://github.com/juangon/liferay-faces-alloy/commit/d8831085c59d68827664724f858b7fc43b4e1818).

This URL should look like this:

    https://github.com/liferay/liferay-portal/blob/eb2dc9930ff437e2b75477632fd06dd1b7932cf0/portal-web/third-party/alloy-2.0.0.67.zip